tfx/components/example_validator/executor.py

```diff
@@ -36,6 +36,8 @@
 STATISTICS_KEY = 'statistics'
 # Key for schema in executor input_dict.
 SCHEMA_KEY = 'schema'
+# Key for training_statistics in executor input_dict.
```

Comment:
nit: pre-existing whitespace is wonky; add a leading newline here and add one before line 37 too

Reply:
Removed whitespace